lib/ansible/plugins/strategy/__init__.py

```diff
@@ -1015,9 +1024,9 @@ def _cond_not_supported_warn(self, task_name):
 
     def _execute_meta(self, task, play_context, iterator, target_host):
 
-        # meta tasks store their args in the _raw_params field of args,
-        # since they do not use k=v pairs, so get that
-        meta_action = task.args.get('_raw_params')
+        # old style meta tasks store their args in the _raw_params field of args,
+        # since they never use to use k=v pairs, now its recomended to use name k=v pair
+        meta_action = task.args.get('_raw_params') or task.args.get('name')
```

Comment:
If we decide to deprecate, then we should have additional logic to detect that the action came in via _raw_params rather than name and issue a deprecation warning.

However, we may want to do this via a "long deprecation cycle" which we've talked about the need for but haven't nailed down any rules for yet.

Reply:
I'm not sure I would want to deprecate the raw_params method, I think it makes sense to support both, especially considering there are other meta actions that do no use any additional params.